go/logic/topology_recovery.go

```diff
@@ -291,6 +294,7 @@ func prepareCommand(command string, topologyRecovery *TopologyRecovery) (result
 	if topologyRecovery.SuccessorKey != nil {
 		command = strings.Replace(command, "{successorHost}", topologyRecovery.SuccessorKey.Hostname, -1)
 		command = strings.Replace(command, "{successorPort}", fmt.Sprintf("%d", topologyRecovery.SuccessorKey.Port), -1)
+		command = strings.Replace(command, "{successorBinlogCoordinates}", topologyRecovery.SuccessorBinlogCoordinates, -1)
```

Comment:
check if binlog coordinates is not nil, and assign `DisplayString()`

Reply:
Thanks for pointing me to that function.